monero-rpc/src/monerod.rs

```diff
@@ -1,46 +1,92 @@
 use anyhow::{Context, Result};
-use serde::Deserialize;
+use monero::{cryptonote::hash::Hash, util::ringct, PublicKey};
+use serde::{de::DeserializeOwned, Deserialize, Serialize, Serializer};
 
 #[jsonrpc_client::api(version = "2.0")]
 pub trait MonerodRpc {
     async fn generateblocks(&self, amount_of_blocks: u32, wallet_address: String)
         -> GenerateBlocks;
     async fn get_block_header_by_height(&self, height: u32) -> BlockHeader;
     async fn get_block_count(&self) -> BlockCount;
+    async fn get_block(&self, height: u32) -> GetBlockResponse;
+    async fn on_get_block_hash(&self, height: [u32; 1]) -> String;
```

Comment:
I cant see this being used anywhere, What does it do?

Reply:
You are right, turned out to not be needed.